server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_1__add_integration_git.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS `integration_git_repository` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `create_time` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `update_time` datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  `creator_id` bigint(20) NOT NULL COMMENT 'creator user id, references iam_user.id',
+  `organization_id` bigint(20) NOT NULL COMMENT 'organization id, references iam_organization.id',
+  `project_id` bigint(20) NOT NULL COMMENT 'project id, references collaboration_project.id',
```

Comment:
a git repo should not belong to any project once created~
in another word, `project_id` can be null

Reply:
done~